RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -863,6 +864,57 @@ def runTauID(self):
             )
             tauIDSources =_tauIDSourcesWithAgainistEle.clone()
 
+        if "newDMPhase2v1" in self.toKeep:
+            if self.debug: print ("Adding newDMPhase2v1 ID")
+            def tauIDMVAinputs(module, wp):
+                return cms.PSet(inputTag = cms.InputTag(module), workingPointIndex = cms.int32(-1 if wp=="raw" else -2 if wp=="category" else getattr(self.process, module).workingPoints.index(wp)))
+            self.process.rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw = patDiscriminationByIsolationMVArun2v1raw.clone(
+                PATTauProducer = cms.InputTag('slimmedTaus'),
```

Comment:
Please drop type specification for the parameters modified in cloned configs, for a safer syntax (see item 4 in https://twiki.cern.ch/twiki/bin/view/CMS/OpenRecoTasks#Review_and_optimization_of_t_AN1)